app/partials/project-modal-editor.cjsx

```diff
@@ -218,6 +218,20 @@ ProjectModalEditorController = React.createClass
     @props.projectModal.update 'steps'
     @props.projectModal.save()
 
+  confirmStepRemove: (index) ->
```

Comment:
You can simplify this function by writing the callback as an anonymous closure, and using the `confirm()` function.
```
confirmStepRemove: (index) ->
      message = "Are you sure you want to delete this step? This action is irreversible!"
      confirm(message, () => @handleStepRemove(index))
```